src/shared/localization.ts

```diff
@@ -19,7 +19,7 @@ export const defaultStrings = {
         bold: shortcut("Bold"),
         code_block: {
             title: shortcut("Code block"),
-            help_text: "Use for code that is more than one line",
+            help_text: "Multiline block of code with syntax highlighting",
```

Comment:
@b-kelly I am not against changing the text per se but in general I would recommend to double check with the author before doing so. Copy is coming from Design.
@shanefoster can you give us a thumbs up about this text change?